### PR TITLE
Config docs

### PR DESCRIPTION
**Pull request by ianlewis**
Update configuration docs; add doc for shim v1; add links to top README

### CODE REVIEW

File: docs/configure-containerd-shim-runsc-v1.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
-This document describes how to configure runtime options for `containerd-shim-runsc-v1`.
-This is follows on to the instructions of [Runtime Handler Quick Start (shim v2) (containerd >=1.2)](runtime-handler-shim-v2-quickstart.md) and requires containerd 1.3 or later.
+This document describes how to configure runtime options for
+`containerd-shim-runsc-v1`.  This is follows on to the instructions of \[Runtime
+Handler Quick Start (shim v2) (containerd
 
-## Configuration
+> =1.2)](runtime-handler-shim-v2-quickstart.md) and requires containerd 1.3 or
+> later.
```
Comment (kevinGC) [inline docs/configure-containerd-shim-runsc-v1.md]: Is the formatting on a link here broken?

File: docs/configure-containerd-shim-runsc-v1.md [hunk 2]
```diff
@@ -24,18 +28,46 @@ EOF
...
+
+The set of options that can be configured can be found in
+[options.go](../pkg/v2/options/options.go).
+
+#### Example: Enable the KVM platform
+
+gVisor enables the use of a number of platforms. This example shows how to
+configure `containerd-shim-runsc-v1` to use gvisor with the KVM platform.
+
+Find out more about platform in teh (gVisor
```
Comment (kevinGC) [inline docs/configure-containerd-shim-runsc-v1.md]: Typo: teh

File: docs/configure-gvisor-containerd-shim.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+`runc_shim`: The path to the runc shim. This is used by the gvisor-containerd-shim to run normal containers.
+`runsc_config`: This is a set of key/value pairs that are converted into `runsc` command line flags. You can learn more about which flags are available by running `runsc flags`.
+
+## Example: Enable the KVM platform
+
+gVisor enables the use of a number of platforms. This configuration enables the
+KVM platform.
+
+Find out more about platform in teh (gVisor
```
Comment (kevinGC) [inline docs/configure-gvisor-containerd-shim.md]: Typo: teh

File: docs/configure-containerd-shim-runsc-v1.md (hunk 1 above)
Comment (ianlewis) [reply docs/configure-containerd-shim-runsc-v1.md]: Done.

File: docs/configure-gvisor-containerd-shim.md (hunk 3 above)
Comment (ianlewis) [reply docs/configure-gvisor-containerd-shim.md]: Done

File: docs/configure-containerd-shim-runsc-v1.md (hunk 2 above)
Comment (ianlewis) [reply docs/configure-containerd-shim-runsc-v1.md]: Done.

**Pull request merged**